### PR TITLE
Add the genai.vector.encode function to the available functions

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-885.surge.sh

Updated pages:
- [Functions](https://neo4j-docs-cypher-885.surge.sh/cypher-manual/5/functions)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

File: modules/ROOT/pages/functions/index.adoc [hunk 1]
```diff
@@ -797,6 +798,18 @@ Database functions provide information about databases.
 label:new[Introduced in 5.12]
 |===
 
+[[header-query-functions-genai]]
+**xref::genai-integrations.adoc#single-embedding[GenAI function]** label:new[Introduced in 5.16] label:aura-only[Available only on Aura]
```
Comment (nilsceberg) [inline modules/ROOT/pages/functions/index.adoc:802]: The heading above says "functions" in the plural even though that also only has one function.

File: modules/ROOT/pages/functions/index.adoc [hunk 2]
```diff
@@ -797,6 +798,18 @@ Database functions provide information about databases.
...
+
+This function is used to generate a vector embedding for a single value.
```
Comment (nilsceberg) [inline modules/ROOT/pages/functions/index.adoc:804]: Should there be a link to the plugin docs?

**Review by nilsceberg — APPROVED** [review]
This looks good, but what do we do when the plugin is released for on-prem users? Simply change "Available only on Aura" to "Requires GenAI plugin" or something?

File: modules/ROOT/pages/functions/index.adoc (hunk 2 above)
Comment (renetapopova) [reply modules/ROOT/pages/functions/index.adoc:804]: This link is to the plugin docs.

File: modules/ROOT/pages/functions/index.adoc (hunk 1 above)
Comment (renetapopova) [reply modules/ROOT/pages/functions/index.adoc:802]: Do you mean the anchor?

**Comment by renetapopova** [comment]
> This looks good, but what do we do when the plugin is released for on-prem users? Simply change "Available only on Aura" to "Requires GenAI plugin" or something?

We could remove the label "Available only on Aura" and add a note that for on-prem they need to install the plugin.

File: modules/ROOT/pages/functions/index.adoc (hunk 2 above)
Comment (renetapopova) [reply modules/ROOT/pages/functions/index.adoc:804]: I mean `xref::genai-integrations.adoc#single-embedding`

Comment (nilsceberg) [reply modules/ROOT/pages/functions/index.adoc:804]: Oh, sorry, I'm just blind!

File: modules/ROOT/pages/functions/index.adoc (hunk 1 above)
Comment (nilsceberg) [reply modules/ROOT/pages/functions/index.adoc:802]: No, I meant that this heading says "GenAI function" but the one above says "Database functions", even though there is only one function under that heading as well. Seemed like they should be consistent in that regard. :)

**Pull request merged**